python_modules/libraries/dagster-azure/dagster_azure/adls2/resources.py

```diff
@@ -48,8 +56,7 @@ def my_job():
     `required_resource_keys`, or it will not be initialized for the execution of their compute
     functions.
 
-    You may pass credentials to this resource using either a SAS token or a key, using
-    environment variables if desired:
+    You may pass credentials to this resource using either a SAS token, a key or by
```

Comment:
is this sentence incomplete?

Reply:
Ah very good catch, I have pushed the rest of the sentence